packages/intl-locale/index.ts

```diff
@@ -640,44 +640,90 @@ export class Locale {
   }
 
   /**
+   * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getCalendars
+   * https://tc39.es/proposal-intl-locale-info/#sec-Intl.Locale.prototype.getCalendars
+   */
+  public getCalendars() {
+    return calendarsOfLocale(this)
+  }
+
+  /**
+   * @deprecated – use local.getCalendars() instead
    * https://tc39.es/proposal-intl-locale/#sec-Intl.Locale.prototype.calendars
    */
   public get calendars() {
-    return calendarsOfLocale(this)
+    return this.getCalendars()
```

Comment:
you should revert this change. This applies to the below implementations as well

Reply:
If both `.calendars`  and `.getCalendars()` are exposed, shouldn't one use the other?

Reply:
No that's not what the spec says